NuGet.Clients.sln

```diff
@@ -121,6 +121,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.SolutionRestoreManage
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.SolutionRestoreManager.Test", "test\NuGet.Clients.Tests\NuGet.SolutionRestoreManager.Test\NuGet.SolutionRestoreManager.Test.csproj", "{F7837EEB-1B49-482F-8722-EB35BA0937CC}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.VisualStudio.Workspace.Extensions.Implementation", "src\NuGet.Clients\NuGet.VisualStudio.Workspace.Extensions.Implementation\NuGet.VisualStudio.Workspace.Extensions.Implementation.csproj", "{CB2D257A-7C56-4B2D-ACC8-BD9114088BC7}"
```

Comment:
I can do that, but I'm of the opinion of having small assemblies which would be loaded as when needed instead of having one large with everything inside.
But if that's the way we are following it in NuGet then I'll merge this with PackageManagement.VisualStudio project